core/kazoo_number_manager/src/kazoo_number_manager_maintenance.erl

```diff
@@ -260,7 +260,7 @@ fix_account_numbers(AccountDb = ?MATCH_ACCOUNT_ENCODED(A,B,Rest)) ->
                             %% timer:sleep(?TIME_BETWEEN_ACCOUNTS_MS),
                             gb_sets:subtract(Leftovers, AuthoritativePNs)
                     end
-                   ,fun (Set1, Set2) -> gb_sets:union(Set1, Set2) end
+                   ,fun (Set1, Set2) -> gb_sets:intersection(Set1, Set2) end
```

Comment:
No need for the fun (not sure why its there); can just be `fun gb_sets:intersection/2`

Reply:
Good catch!